bubblewrap.c

```diff
@@ -520,6 +520,11 @@ monitor_child (int event_fd, pid_t child_pid, int setup_finished_fd)
   assert (j < sizeof(dont_close)/sizeof(*dont_close));
   fdwalk (proc_fd, close_extra_fds, dont_close);
 
+  struct sigaction sa;
+  sigemptyset (&sa.sa_mask);
+  sa.sa_handler = SIG_DFL;
+  sigaction(SIGCHLD, &sa, NULL);
```

Comment:
If we need this workaround, should it be reset in `main()` instead (before we fork), so that children of `bwrap` (the *COMMAND* from its command-line) can also rely on `SIGCHLD` having its normal behaviour?

Reply:
Ok, move the signal reset to start doesn't affect the expected behavior.